crypto/err/openssl.txt

```diff
@@ -696,6 +696,7 @@ EC_F_EC_POINT_INVERT:210:EC_POINT_invert
 EC_F_EC_POINT_IS_AT_INFINITY:118:EC_POINT_is_at_infinity
 EC_F_EC_POINT_IS_ON_CURVE:119:EC_POINT_is_on_curve
 EC_F_EC_POINT_MAKE_AFFINE:120:EC_POINT_make_affine
+EC_F_EC_POINT_MUL:309:
```

Comment:
@mattcaswell out of curiosity, what is going on here after `make update`? I'm not great with the error side of OpenSSL, but looking at the diff I'd expect the function name on the RHS.

Reply:
OpenSSL no longer displays the function name, so that field is now blank.